backend/src/src-licenseinfo/src/main/java/com/siemens/sw360/licenseinfo/parsers/SPDXParser.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright Bosch Software Innovations GmbH, 2016.
+ * Part of the SW360 Portal Project.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package com.siemens.sw360.licenseinfo.parsers;
+
+import com.google.common.collect.ImmutableList;
+import com.siemens.sw360.datahandler.couchdb.AttachmentConnector;
+import com.siemens.sw360.datahandler.thrift.SW360Exception;
+import com.siemens.sw360.datahandler.thrift.attachments.Attachment;
+import com.siemens.sw360.datahandler.thrift.attachments.AttachmentContent;
+import com.siemens.sw360.datahandler.thrift.licenseinfo.LicenseInfo;
+import com.siemens.sw360.datahandler.thrift.licenseinfo.LicenseInfoParsingResult;
+import com.siemens.sw360.datahandler.thrift.licenseinfo.LicenseInfoRequestStatus;
+import org.apache.log4j.Logger;
+import org.spdx.rdfparser.InvalidSPDXAnalysisException;
+import org.spdx.rdfparser.SPDXDocumentFactory;
+import org.spdx.rdfparser.model.SpdxDocument;
+
+import java.io.File;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Function;
+
+/**
+ * @author: alex.borodin@evosoft.com
+ * @author: maximilian.huber@tngtech.com
+ */
+public class SPDXParser extends LicenseInfoParser {
+    public static final String FILETYPE_SPDX_EXTERNAL = "SPDX";
+    protected static final String FILETYPE_SPDX_INTERNAL = "RDF/XML";
+    private static final List<String> ACCEPTABLE_ATTACHMENT_FILE_EXTENSIONS = ImmutableList.of(
+            "rdf",
+            "spdx"
+    );
```

Comment:
I am not sure what you mean, that are the possible file extensions a spdx-document could have. Even `.spdx` might have to be removed, since tag/value is not yet parseable.